xclim/core/datachecks.py

```diff
@@ -59,3 +59,40 @@ def check_daily(var: xr.DataArray):
     This does not check for gaps in series.
     """
     return check_freq(var, "D")
+
+
+@datacheck
+def check_common_time(*inputs):
+    """Raise an error if the list of inputs doesn't have a single common frequency.
+
+    This raises if:
```

Comment:
I would use the numpy docstring `raises` convention here: https://numpydoc.readthedocs.io/en/v1.5.0/format.html#raises